ssl/t1_lib.c

```diff
@@ -1359,6 +1379,40 @@ int ssl_add_clienthello_tlsext(SSL *s, WPACKET *pkt, int *al)
         return 0;
     }
 
+
+    /* TODO(TLS1.3): Should we add this extension for versions < TLS1.3? */
```

Comment:
I guess it might be more compatible to not send it, so I think you should just remove that TODO.